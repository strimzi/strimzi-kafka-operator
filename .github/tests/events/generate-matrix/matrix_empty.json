{
  "event_name": "workflow_dispatch",
  "action": "workflow_dispatch", 
  "inputs": {
    "pipelines": "",
    "profiles": ""
  },
  "repository": {
    "name": "ci-playground",
    "owner": { "login": "frawless" }
  }
}