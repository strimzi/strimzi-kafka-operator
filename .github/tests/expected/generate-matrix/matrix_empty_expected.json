[
  {
    "agent": "oracle-4cpu-16gb-x86-64",
    "arch": "amd64",
    "pipeline": "custom",
    "profile": "all",
    "timeout": 720,
    "strimzi_feature_gates": "",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "bundle",
    "parallel": "1",
    "groups": "",
    "tests": "",
    "jobName": "custom-all-amd64"
  }
]